awx/main/conf.py

```diff
@@ -126,6 +126,15 @@ def _load_default_license_from_file():
     category_slug='system',
 )
 
+register(
+    'INSTALL_UUID',
+    field_class=fields.CharField,
+    label=_('Unique identifier for an AWX/Tower installation'),
+    category=_('System'),
+    category_slug='system',
+    read_only=True,
```

Comment:
@wenottingham I took another pass at this, and for now, I'm making it read-only (because I want to protect from accidental deletes, and enforcing a value is trickier for `$reasons`).